receiver/routes/setup.py

```
@@ -689,7 +693,11 @@ def get_retention():
         dns_source = 'default'
 
     # Estimate log counts for slider steps
-    estimates = _estimate_log_counts()
+    try:
+        estimates = _estimate_log_counts()
+    except Exception as exc:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)